src/common/common.css

```diff
@@ -71,12 +71,22 @@ body {
 .success::before {
   background-image: url('/common/img/check.svg');
 }
+
 .icon-dismiss {
   background-image: url('/common/img/close.svg');
   position: absolute;
-  right: 10px;
+  right: 0px;
   cursor: pointer;
 }
+.icon-dismiss:hover {
+  background-color: rgba(12, 12, 13, 0.1);
+}
+.icon-dismiss:active {
+  background-color: rgba(12, 12, 13, 0.2);
+}
+.icon-dismiss:focus {
```

Comment:
I just tested around a bit and the element is not even focusable by default. You need to add `tabindex="1"` to make it actually focusable. Then we can see the design.

Also, I think here yet again, a rounded border should be used (4px), although not explicitly specified in the Photon guide, but the screenshots look like this.

Reply:
Isn't this documented [here](https://design.firefox.com/photon/components/buttons.html#default-3)? There they say "Corner Radius: `2px`"

Reply:
Hmm, only for the default (big) button as far as I see, nothing is said for ghost buttons.
But yes, maybe use 2px, because if the other buttons have two pixels then this one may have, too.